src/Service/Startup.cs

```diff
@@ -798,8 +798,16 @@ private void ConfigureAuthentication(IServiceCollection services, RuntimeConfigP
                         }
                     }
 
-                    services.AddAuthentication(EasyAuthAuthenticationDefaults.AUTHENTICATIONSCHEME)
+                    string scheme = easyAuthType switch
+                    {
+                        EasyAuthType.AppService => EasyAuthAuthenticationDefaults.APPSERVICEAUTHSCHEME,
+                        EasyAuthType.StaticWebApps => EasyAuthAuthenticationDefaults.SWAAUTHSCHEME,
+                        _ => EasyAuthAuthenticationDefaults.AUTHENTICATIONSCHEME
+                    };
+
+                    services.AddAuthentication(scheme)
```

Comment:
Previously, when SWAAUTHSCHEME was the scheme, we did not need to provide that as the value to the AddAuthentication function. the generic value of EasyAuthAuthenticationDefaults.AUTHENTICATIONSCHEME = EasyAuthAuthentication was sufficient. What changed with AppService, that we need to specify the explicit value of the scheme to the AddAuthentication function?